docs/how-to-guides/performing-cluster-maintenance.md

```diff
@@ -0,0 +1,239 @@
+microceph.mon         enabled   active    -
+microceph.osd         disabled  inactive  -
+microceph.rbd-mirror  disabled  inactive  -
+microceph.rgw         disabled  inactive  -
+
+$ juju ssh microceph/3 -- sudo microceph.ceph -s
+  cluster:
+    id:     156f1fa0-51a5-4515-a538-2fe32a8803b5
+    health: HEALTH_WARN
+            noout flag(s) set
+            1 osds down
+            1 host (1 osds) down
+            Degraded data redundancy: 2/6 objects degraded (33.333%), 1 pg degraded, 1 pg undersized
+
+  services:
+    mon: 4 daemons, quorum juju-681fff-1,juju-681fff-0,juju-681fff-2,juju-681fff-3 (age 25m)
+    mgr: juju-681fff-1(active, since 25m), standbys: juju-681fff-0, juju-681fff-2
+    osd: 4 osds: 3 up (since 100s), 4 in (since 24m)
+         flags noout
+
+  data:
+    pools:   1 pools, 1 pgs
+    objects: 2 objects, 449 KiB
+    usage:   107 MiB used, 16 GiB / 16 GiB avail
+    pgs:     2/6 objects degraded (33.333%)
+             1 active+undersized+degraded
+```
+
+For reference, comparing the status of the cluster **before** entering maintenance mode:
+
+```text
+$ juju ssh microceph/3 -- sudo snap services microceph
+Service               Startup   Current   Notes
+microceph.daemon      enabled   active    -
+microceph.mds         disabled  inactive  -
+microceph.mgr         disabled  inactive  -
+microceph.mon         enabled   active    -
+microceph.osd         enabled   active    -
+microceph.rbd-mirror  disabled  inactive  -
+microceph.rgw         disabled  inactive  -
+
+$ juju ssh microceph/3 -- sudo microceph.ceph -s
+  cluster:
+    id:     156f1fa0-51a5-4515-a538-2fe32a8803b5
+    health: HEALTH_OK
+
+  services:
+    mon: 4 daemons, quorum juju-681fff-1,juju-681fff-0,juju-681fff-2,juju-681fff-3 (age 21m)
+    mgr: juju-681fff-1(active, since 22m), standbys: juju-681fff-0, juju-681fff-2
+    osd: 4 osds: 4 up (since 20m), 4 in (since 20m)
+
+  data:
+    pools:   1 pools, 1 pgs
+    objects: 2 objects, 449 KiB
+    usage:   107 MiB used, 16 GiB / 16 GiB avail
+    pgs:     1 active+clean
+```
+
+Note that the `microceph.osd` service is disabled and inactive after entering maintenance mode; the cluster also has
+noout flag set.
+
+### 2. Exit maintenance mode for microceph node
+
+To recover unit `microceph/3` from maintenance mode, run
+
+```shell
+juju run microceph/3 exit-maintenance
+```
+
+Sample output is:
+
+```text
+$ juju run microceph/3 exit-maintenance
+Running operation 17 with 1 task
+  - task 18 on unit-microceph-3
+
+Waiting for task 18...
+actions:
+  step-1:
+    description: Run `ceph osd unset noout`.
+    error: ""
+    id: unset-noout-ops
+  step-2:
+    description: Assert osd has 'noout' flag unset.
+    error: ""
+    id: assert-noout-flag-unset-ops
+  step-3:
+    description: Start osd service in node 'juju-681fff-3'.
+    error: ""
+    id: start-osd-ops
+errors: ""
+status: success
+```
+
+After exiting maintenance mode, the status of the cluster is:
```

Comment:
"This is the cluster status after exiting maintenance node for unit `microceph/3`"